### ISSUE TITLE
Bug in the soundspeed calculation 

### ISSUE THREAD

**Issue by justinread** [pynbody/pynbody#304]
Hi, 

I think the gas sound speed calculation in PyNbody is currently not quite correct. The code is:

@SimSnap.derived_quantity
[docs]def cs(self):
    """Sound speed"""
    return np.sqrt(5.0 * units.k * self['temp'] / self['mu'] / units.m_p)

I think the 5.0 should be 5.0 / 3.0 - i.e. "gamma". 

I'm happy to put this through a pull request but just wanted to check here first that there isn't some reason that it should be 5.0. Am I missing something? 

Justin

**Comment by trquinn** [comment]
I just ran into this bug.  I concur that it should be "5.0/3.0", and perhaps documentation saying that it is for gas with no internal degrees of freedom.

**Comment by apontzen** [comment]
I agree... @justinread do you have a patch? If so could you open a pull request?

**Comment by biernackip** [comment]
I have submitted a fix for this, as no one wanted to take care of it [(https://github.com/pynbody/pynbody/pull/406)](https://github.com/pynbody/pynbody/pull/406)

**Comment by apontzen** [comment]
Thanks! I'll merge as soon as the tests pass
